### PR TITLE
Fix Part of #2833 : Add missing named argument in StoryFragmentTest

### PR DESCRIPTION
**Pull request by ChiragKalra**
## Explanation

Fixes Part of #2833, Added Missing Named Argument in StoryFragmentTest

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
-  [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt [hunk 1]
```diff
@@ -460,7 +464,11 @@ class StoryFragmentTest {
       testCoroutineDispatchers.runCurrent()
       onView(isRoot()).perform(orientationLandscape())
       val headerString: String =
-        getResources().getQuantityString(R.plurals.story_total_chapters, 2, 1, 2)
+        getResources().getQuantityString(
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt]: Can we keep this in one line?

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt [hunk 2]
```diff
@@ -487,25 +495,25 @@ class StoryFragmentTest {
...
+      internalProfileId = internalProfileId,
+      topicId = TEST_TOPIC_ID_1,
+      storyId = TEST_STORY_ID_1
     )
   }
 
   private fun setStoryPartialProgressForFractions() {
     storyProgressTestHelper.markCompletedFractionsStory0Exp0(
-      profileId,
+      profileId = profileId,
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt]: we can skip this

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt [hunk 3]
```diff
@@ -487,25 +495,25 @@ class StoryFragmentTest {
...
   private fun createTestStoryActivityIntent(): Intent {
     return StoryActivity.createStoryActivityIntent(
-      ApplicationProvider.getApplicationContext(),
-      internalProfileId,
-      TEST_TOPIC_ID_1,
-      TEST_STORY_ID_1
+      context = ApplicationProvider.getApplicationContext(),
+      internalProfileId = internalProfileId,
+      topicId = TEST_TOPIC_ID_1,
+      storyId = TEST_STORY_ID_1
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt]: we can skip for this function

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt [hunk 4]
```diff
@@ -487,25 +495,25 @@ class StoryFragmentTest {
...
   private fun createFractionsStoryActivityIntent(): Intent {
     return StoryActivity.createStoryActivityIntent(
-      ApplicationProvider.getApplicationContext(),
-      internalProfileId,
-      FRACTIONS_TOPIC_ID,
-      FRACTIONS_STORY_ID_0
+      context = ApplicationProvider.getApplicationContext(),
+      internalProfileId = internalProfileId,
+      topicId = FRACTIONS_TOPIC_ID,
+      storyId = FRACTIONS_STORY_ID_0
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt]: we can skip this for this function

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @ChiragKalra 
suggested nit changes

**Comment by ChiragKalra** [comment]
> suggested nit changes

Applying changes and pushing.

**Review by anandwana001 — APPROVED** [review]
LGTM, Also, the conversation should be resolve only by the author who has started the conversation.

**Comment by ChiragKalra** [comment]
> LGTM, Also, the conversation should be resolve only by the author who has started the conversation.

Sorry, will keep that in mind next time.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks

**Pull request merged**